source/frontend/src/features/contacts/contact/edit/Person/UpdatePersonForm.tsx

```diff
@@ -90,6 +91,9 @@ const UpdatePersonComponent: React.FC<
   };
 
   const isContactMethodInvalid = useMemo(() => {
+    if (isPersonDisabled === 'true') {
```

Comment:
one is a boolean, the right hand is a string. probably need to check the right type?